src/components/inputCheckbox/AposCheckbox.vue

```diff
@@ -47,7 +47,7 @@ export default {
     },
     id: {
       type: String,
-      required: true
+      default: null
```

Comment:
This is because the media manager uses the raw checkbox without an id. Maybe there should be one?